mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Requests tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+
+  Scenario: Given an item Id, a user Id, and a pickup location, attempt to create a page request when the applicable request policy disallows pages
+    * def itemId = call uuid1
+    * def servicePointId = call uuid1
+    * def userId = call uuid1
+    * def userBarcode = 'FAT-1030UBC'
+    * def itemBarcode = 'FAT-1030IBC'
+    * def groupId = call uuid1
+    * def instanceId = call uuid1
+    * def locationId = call uuid1
+    * def holdingId = call uuid1
+    * def materialTypeId = call uuid1
```

Comment:
based on my experience it would be better to move it into background section

Reply:
done

Reply:
I mean only materialTypeId

Reply:
haha, but you selected bunch of them)